integration-cli/docker_cli_ps_test.go

```diff
@@ -245,6 +244,18 @@ func (s *DockerSuite) TestPsListContainersFilterStatus(c *check.C) {
 		c.Fatalf("Expected error response due to invalid status filter output: %q", out)
 	}
 
+	// pause running container
+	out, _ = dockerCmd(c, "run", "-itd", "busybox")
+	pausedID := strings.TrimSpace(out)
+	dockerCmd(c, "pause", pausedID)
+	// make sure the container is unpaused to let the daemon stop it properly
+	defer func() { dockerCmd(c, "unpause", pausedID) }()
+
+	out, _ = dockerCmd(c, "ps", "-q", "--filter=status=paused")
```

Comment:
if you use `--no-trunc` you don't need truncate with `[:12]` and it's more robust